<launch>
    <arg name="points_parent_src" default="/vlp16_left/velodyne_points" />
    <arg name="points_child_src" default="/vlp16_right/velodyne_points" />
    <arg name="log_file_path" default="/home/zhaolei/.ros/A.txt" />


    <node pkg="multi_lidar_calibrator" type="joint_two_pointclouds" name="joint_two_pointclouds" output="screen">
        <param name="points_parent_src" value="$(arg points_parent_src)" />
        <param name="points_child_src" value="$(arg points_child_src)" />
        <param name="log_file_path" value="$(arg log_file_path)" />

    </node>

</launch>
